Dynatrace/dynatrace-operator
# Add suggested namespace annotation for OLM

# Description

Currently OLM shows only openshift-operators namespace as choice, if you install the operator through their marketplace. However there is a workaround for that by adding `operatorframework.io/suggested-namespace: dynatrace` this annotation. It allows the user to also choose the dynatrace namspace as installation target

## How can this be tested?

1. Create a new Bundle and deploy it to you openshift cluster
2. Test if the dynatrace namespace is a choice in the namespace dropdown, when you install your operator
3. Test if everything works as expected (classicFullStack, applicationMonitoring without CSI Driver)


## Checklist
